rust/chains/tw_solana/src/modules/compiled_keys.rs

```diff
@@ -81,18 +81,21 @@ impl CompiledKeys {
             (meta.is_signer == is_signer && meta.is_writable == is_writable).then_some(*account)
         };
 
-        let writable_signer_keys: Vec<SolanaAddress> = ordered_keys
+        let mut writable_signer_keys: Vec<SolanaAddress> = ordered_keys
             .iter()
             .filter_map(|key| filter(key, true, true))
             .collect();
-        let readonly_signer_keys: Vec<SolanaAddress> = ordered_keys
+        writable_signer_keys.sort();
```

Comment:
Could you please revert the keys sorting? Even though, they're sorted in official Solana lib, but in WalletCore I disabled it explicitly to save backward compatibility to have the same transaction signature after the Rust transition.
Please note that the keys are not required to be sorted to be valid transaction